src/app/containers/FrontPageMain/index.jsx

```diff
@@ -1,11 +1,9 @@
 /* eslint-disable jsx-a11y/aria-role */
 import React, { useContext } from 'react';
-import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import path from 'ramda/src/path';
 import { frontPageDataPropTypes } from '#models/propTypes/frontPage';
-import { Grid, GridItemConstrainedLargeWithTopMargin } from '#lib/styledGrid';
 import { ServiceContext } from '#contexts/ServiceContext';
-import FrontPageSection from '../FrontPageSection';
+import Index from '../Index';
```

Comment:
`Index` seems very generic to me. Should we call it `PageMain` and also include the `main` tag inside?

Reply:
`Index` seemed to be the term people were using for a list of story promos, but I'll reconsider when I look at this again. I think `PageMain` might be more confusing as it could be any page

Reply:
I believe that the word `Index` is used for any page containing a collection of stories. So anything from a front page to a feature index to a topic page, as long as it contains a list of stories, rather than a single story (like an articles page or a media page) is considered an `Index`